core/src/main/resources/schema/app/20201221232206_CB-9746:_Use_new_loadbalancer_endpoints_for_DL_services.sql

```diff
@@ -0,0 +1,10 @@
+-- // CB-9746: Use new loadbalancer endpoints for DL services
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE loadbalancer ADD COLUMN IF NOT EXISTS fqdn character varying(255) DEFAULT 'No Info';
+UPDATE loadbalancer SET fqdn = 'No Info' WHERE fqdn IS NULL;
```

Comment:
I don't think we should have the default be a string value. You're not checking the value of the load balancer FQDN name anywhere that I saw, so it seems like the code could see the string "No Info" and think that's a perfectly valid domain. I don't think we need a default value, since if it isn't set it should just be null, and then the statements that check if it's null will correctly evaluate to false.